Firestore/core/src/firebase/firestore/remote/remote_store.h

```diff
@@ -66,7 +66,7 @@ NS_ASSUME_NONNULL_BEGIN
  * backend for any active target.
  *
  * @param targetID The targetID corresponding to a listen initiated via
- *     -listenToTargetWithQueryData: on FSTRemoteStore.
+ *     a `Listen` on `RemoteStore`.
```

Comment:
At this point, you could "rephrase" to "... initiated via `RemoteStore::Listen()`." or similar.

Reply:
Done.